js/exercise-01.js

```diff
@@ -0,0 +1,24 @@
+/* 
+    1. Create a variable called 'name' that points at a string containing your name
+    2. Define function 'updateValue', which receives a value and updates variable 'name'
+    3. On the next line, call function 'updateValue' with your favourite musician's name
+      Log 'name' value.
+    4. On the next line, call function 'updateValue' with value 'true'
+      Log 'name' value.
+    5. On the next line, call function 'updateValue' value '28'
+      Log 'name' value.
+
+*/
+
+let userName = name;
+
+function updateValue(newName){
+  userName = newName;
+}
+
+updateVaule("Hard GZ");
```

Comment:
Typo
```suggestion
updateValue("Hard GZ");
```

Are you sure this script is working as expected? 🤔

FIX: analyse and fix script if required
FIX: fix typo